lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/TestClassLoaders.java

```diff
@@ -20,36 +20,40 @@
 
 import org.apache.reef.tang.exceptions.BindException;
 import org.apache.reef.tang.exceptions.InjectionException;
-import org.apache.reef.tang.exceptions.NameResolutionException;
 import org.apache.reef.tang.types.ClassNode;
 import org.junit.Assert;
+import org.junit.Test;
 
 import java.io.File;
+import java.io.FilenameFilter;
 import java.net.MalformedURLException;
 
 public class TestClassLoaders {
 
-  //  @Test
-  public void testOneJar() throws MalformedURLException,
-      ClassNotFoundException, NameResolutionException, BindException {
+  private static final FilenameFilter JAR_FILE_FILTER = new FilenameFilter() {
+    public boolean accept(final File dir, final String name) {
+      return name.endsWith(".jar");
```

Comment:
I don't think using hardcoded version hurts maintainability (in this particular case), since we update versions automatically and all at once. I'd still prefer hardcoded jar names, since they are different for different tests in this file.